### ISSUE TITLE
systemd fails with dune exec

### ISSUE THREAD

**Issue by mreppen** [Drup/fugit#2]
When running `dune exec src/fugit.exe`, it fails with
```Failed to find executable src/fugit.exe: No such file or directory```

This is due to `Bos.OS.exec`, which is `Sys.argv.(0)`, which is `src/fugit.exe`.

Running with dune is perhaps not the normal use case, but could this fail in other scenarios? Would `Sys.executable_name` be a better alternative?

**Comment by Drup** [comment]
Yeah, this is all quite annoying. I'm not sure how to retrieve the actual path to the binary currently executed. Note that calling through dune is particularly ill-advised here, since the executable would disappear after a `clean`.

Maybe it's a good argument for just calling `notify-send` instead of `fugit ding` in the systemd action, but that has several other disadvantages.

**Comment by mreppen** [comment]
My understanding is that `Sys.executable_name` is precisely the path to the executing binary. Some quick tests shows it gives the right result also with dune.
